source/renderer/app/api/api.js

```diff
@@ -1194,6 +1207,10 @@ const _createWalletFromServerData = action(
       balance.total.unit === 'lovelace'
         ? new BigNumber(balance.total.quantity).dividedBy(LOVELACES_PER_ADA)
         : new BigNumber(balance.total.quantity);
+    const walletAvailableAmount =
```

Comment:
Any reason to use different variable name than `amount` here?
If variables `walletTotalAmount`, `walletAvailableAmount`, and `walletRewardAmount` become to `amount`, `availableAmount`, `reward`, then you can use variable keys only when returning new Wallet in line 1219.

Reply:
@yakovkaravelov I understand what is your thought but I'd like to keep this as is because it is more descriptive what we are passing to Wallet class. What you thing?

Reply:
@tomothespian let's keep it as is 🙏